docs/webhooks.rst

```diff
@@ -0,0 +1,10 @@
+Registering webhooks to be notified of helpesk events
+-----------------------------------------------------
+
+You can register webhooks to allow third party apps to be notified of helpdesk events. Webhooks can be registered in one of two ways:
+
+1. Setting comma separated environement variables; ``HELPDESK_NEW_TICKET_WEBHOOK_URLS``& ``HELPDESK_FOLLOWUP_WEBHOOK_URLS``.
```

Comment:
This statement is unclear what it means. It would be best to say a comma separate list of webhook URL's for each of the environment variables.
If you do not cater for a space in the URL list then it should be stated here that there must be no spaces before or after commas.